lib/portManager.ts

```diff
@@ -7,15 +7,12 @@ import {
 } from '../utils/PortManagerServer';
 import { PORT_MANAGER_SERVER_PORT } from '../constants/ports';
 import { RequestPortsData } from '../types/PortManager';
-import { detectPort } from '../utils/detectPort';
 import { logger } from './logger';
 
 export const BASE_URL = `http://localhost:${PORT_MANAGER_SERVER_PORT}`;
 
 export async function isPortManagerPortAvailable(): Promise<boolean> {
```

Comment:
Need to keep this around for backwards compatibility reasons since we use it in the CLI, but I can't just import it to `PortManagerServer` because it would create a cyclical dep.